### ISSUE TITLE
[Feature] <Docker container>

### ISSUE THREAD

**Issue by etpeterson** [OSIPI/TF2.4_IVIM-MRI_CodeCollection#64]
### Feature description

Dockerize the code so that processing could be run from the command line, e.g. docker run <TF2.4image> <IVIM input images> <IVIM output images>

### Describe the solution

This would simply put the OSIPI code inside of a docker container that could be run on the command line. Something like this for now: https://stackoverflow.com/questions/42625947/docker-input-output-outside-the-container

Just to run one and exit. Future work would be to keep it alive for future connections.

### Describe alternatives

The current approach doesn't use the docker container. This would allow for someone to run the processing without installing anything other than docker and downloading our image. Further work could make it more of a server architecture, but this is the first step.

Other containerization could be used as well, such as singularity. Perhaps both could be supported in the future.

### Are you working on this?

None

**Comment by Unique-Usman** [comment]
@etpeterson, following up with this, I have some questions.

1. By dockerizing the code, we are basically dockerizing this wrap script created in https://github.com/OSIPI/TF2.4_IVIM-MRI_CodeCollection/issues/58?
2. This is regarding how the input and the output data will be mounted when running the docker image, I am thinking bind mounting might be better since the input and output might keep changing, and I do not think we have all the input data in the repository as some are gotten from zenodo. What do you think ?

**Comment by etpeterson** [comment]
Yes, I think you're correct for both. However, the data should be provided and not downloaded from zenodo. We can assume people will send data in - essentially point to some files or folders. In fact that should be a requirement.
